docker-compose.yaml

```diff
@@ -61,8 +69,35 @@ services:
       retries: 50
     restart: always
 
-  airflow-webserver:
+  # `shared_model_common` and `airflow_common` are not deployed, but they are 
+  # used to push the common airflow image to docker hub
+  shared_model_common:
+    profiles: ["pushable"]
+    image: kboiiz/air-quality-shared-models:latest
+    build: ./services/shared_models
+  airflow-common:
     image: kboiiz/air-quality-airflow:latest
+    profiles: ["pushable"]
+    build: ./services/etl
+    env_file: ./.env
+    environment:
+      AIRFLOW__CORE__EXECUTOR: CeleryExecutor
+      AIRFLOW__CORE__DAGS_ARE_PAUSED_AT_CREATION: 'true'
+    user: "${AIRFLOW_UID:-1000}:${AIRFLOW_GID:-0}"
+    volumes:
+      - ./dags:/opt/airflow/dags
+      - ./logs:/opt/airflow/logs
+      - ./plugins:/opt/airflow/plugins
+      - ./services/shared_models:/opt/airflow/dags/shared_models
+    depends_on:
+      redis:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+  
+  airflow-webserver:
+    profiles: ["airflow", "full", "debug", "pushable"]
```

Comment:
I think these specific airflow services shouldn't have the "pushable" profile tag right?